.github/workflows/tutorials.yml

```diff
@@ -74,49 +74,42 @@ jobs:
         fi
         # dask error on osx, skip dask tuto
         echo "::set-env name=SKIP::--deselect=examples/seismic/tutorials/04_dask.ipynb"
-
     - name: Build docker image
       if: matrix.name == 'tutos-docker-gcc-py36'
       run: |
           docker build . --file docker/Dockerfile --tag devito_img
           echo "::set-env name=RUN_CMD::docker run --rm --name testrun devito_img"
-
     - name: Install dependencies
       if: matrix.name != 'tutos-docker-gcc-py36'
       run: |
         python -m pip install --upgrade pip
         pip install -e .
         pip install matplotlib blosc
-
```

Comment:
Seems some formatting has gone awry here, so please re-add the spaces etc.

Reply:
Done!